tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -11,98 +10,266 @@
     preparsing_validators=[
         validators.hasLength(156),
     ],
-    postparsing_validators=[],
+    postparsing_validators=[
+      validators.if_then_validator(
+          condition_field='RECEIVES_FOOD_STAMPS', condition_function=validators.matches(1),
```

Comment:
OFA is having a mtg next week about whether we are going to keep receipt status items, in light of some recent feedback from STTs on new instructions. should have a decision before this qasp review is complete. if we drop receipt status items, then we can drop these validation checks. else, we need to check the logic in both directions (i.e. also if amount > 0, then receipt == 1:Yes, & if receipt ==1:Yes, then amt >0).

Reply:
OFA has decided to drop this cat3 check.